project_euler/problem_145/sol1.py

```diff
@@ -13,44 +13,81 @@
 
 How many reversible numbers are there below one-billion (10^9)?
 """
+EVEN_DIGITS = [digit for digit in range(10) if digit % 2 == 0]
+ODD_DIGITS = [digit for digit in range(10) if digit % 2 == 1]
 
 
-def odd_digits(num: int) -> bool:
+def reversible_numbers(
+    remaining_length: int, remainder: int, digits: list, length: int
+) -> int:
     """
-    Check if the number passed as argument has only odd digits.
-    >>> odd_digits(123)
-    False
-    >>> odd_digits(135797531)
-    True
+    Count the number of reversible numbers of given length.
+    >>> reversible_numbers(1, 0, [None], 1)
```

Comment:
In the function signature, you've specified the third parameter type as `list`. Is it a list of integers or list of `None`? As per the examples in the docstring, it seems to be a list of `None` but the parameter name is "digits" which suggests it's a list of integers.

Reply:
It is a list of integers. Clarified parameter type specification and replaced `None` with zero.